src/test/java/org/jenkinsci/plugins/gitclient/GitClientTest.java

```diff
@@ -258,6 +259,32 @@ public void setGitClient() throws IOException, InterruptedException {
         gitCmd.run("config", "user.email", "email.from.git.client@example.com");
     }
 
+    private boolean fileProtocolIsAllowed = false;
+
+    /**
+     * Removes the protocol.file.allow setting from the global git
+     * configuration of the current user.
+     */
+    @After
```

Comment:
Better to factor this out into a common method, even a `TestRule`.

Reply:
I'll see if I can find a way to factor it into a common location.  

I've never written a `TestRule` before and I'm running out of weekend time that I can invest, so if a `TestRule` is needed, then I'll likely need to wait a few weeks before I can address the issue.  Next weekend and the following weekend will be filled with watching the grandchildren while their parents help a Lego League event.

Reply:
https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExternalResource.html is the easiest example. Just a suggestion of an idiomatic way to remove the duplicated code.